### ISSUE TITLE
SRU 

### ISSUE THREAD

**Issue by MLdoesML** [etheleon/deepore#4]
Implementing SRU u first because it is easier and will speed up training

**Comment by MLdoesML** [comment]
Implemented SRU on the SRU branch, build ontop of the weight visualisation branch.

**Comment by MLdoesML** [comment]
However the SRU don't have batch norm which the paper uses with it's LSTM

**Comment by MLdoesML** [comment]
Ok implemented Batch normalisation
